evaluation_script/main.py

```diff
@@ -1,5 +1,46 @@
-import os
-import gzip
+from utils import get_file_extension
+from utils import open_file
+from utils import calculate_precision
+from utils import calculate_recall
+from utils import calculate_fpr
+from utils import calculate_f1
+from utils import BaseMetrics
+from utils import Metrics
+
+query_level_base_metrics = {}
```

Comment:
A bit cleaner would be to create a new instance each invocation of evaluation. The advantage being that evaluation could be called multiple times in the same python invocation. At the moment though multiple calls to evaluate will get incorrect results because these are not being reset. In fact, the unit tests may fail due to this.

The simple thing to do without doing further restructuring is to clear these at the start of the evaluate function, similar to how the `tp`, `fp`, etc variables are getting set to zero.

Reply:
Fixed in latest version.